src/signals/incident/components/form/MapInput/index.js

```diff
@@ -2,17 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import mapLocation from 'shared/services/map-location';
-import MapInteractive from 'components/MapInteractive';
+import MapInteractive from 'components/MapEditor';
 
+import MAP_OPTIONS, { DEFAULT_MARKER_POSITION } from 'shared/services/configuration/map-options';
 import Header from '../Header';
 
 const MapInput = ({
   handler, touched, hasError, meta, parent, getError, validatorsOrOpts,
 }) => {
-  const value = handler().value || {};
+  const value = handler().value || {
+    geometrie: {
+      type: "Point",
+      coordinates: [DEFAULT_MARKER_POSITION.lng, DEFAULT_MARKER_POSITION.lat],
```

Comment:
Why would we need to render a marker when there is no location available? Doesn't seem right.

Reply:
This is just for testing this pr. Will be removed in the next pr.